robot/rover/MobilePlatform/dev/pure_pursuit_followerCMakeLists.txt

```diff
@@ -0,0 +1 @@
+fuck you
```

Comment:
Honestly I think some of these dev projects should just be removed @vashmata @PeterGhimself